common/lib/xmodule/xmodule/contentstore/mongo.py

```diff
@@ -247,19 +247,64 @@ def _get_all_content_for_course(self,
             contentType: The mimetype string of the asset
             md5: An md5 hash of the asset content
         '''
-        query = query_for_course(course_key, "asset" if not get_thumbnails else "thumbnail")
-        find_args = {"sort": sort}
-        if maxresults > 0:
-            find_args.update({
-                "skip": start,
-                "limit": maxresults,
-            })
+        # TODO: Using an aggregate() instead of a find() here is a hack to get around the fact that Mongo 3.2 does not
```

Comment:
Can you add the ticket number to this comment so that when it happens we'll easily be able to find it?